packages/jest-types/__typetests__/expect.test.ts

```diff
@@ -214,55 +214,65 @@ expectError(expect(jest.fn()).toHaveBeenCalledTimes());
 expectType<void>(expect(jest.fn()).toBeCalledWith());
 expectType<void>(expect(jest.fn()).toBeCalledWith('value'));
 expectType<void>(expect(jest.fn()).toBeCalledWith('value', 123));
+expectType<void>(
```

Comment:
great you've added tests that catch this 👍

Reply:
I was sure you wouldn’t accept this PR without these tests (;

Reply:
😀